CHANGELOG.md

```diff
@@ -12,7 +12,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* None.  
+* Generate the correct LD `-l` flags for XCFrameworks containing libraries
```

Comment:
nit: can you add two empty spaces at the end of this sentence for markdown formatting?